### PR TITLE
Feature/player 542

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Add constants for "Audio" and "Subtitles" to  language files

### CODE REVIEW

**Comment by jpfranco** [comment]
Please take a look at the DOC-3771 ticket. We will mostly likely have to request translations for these new texts.

File: languageFiles/ko.json [hunk 1]
```diff
@@ -132,5 +132,7 @@
   "Closed Captions":"선택 자막",
   "Full Screen": "전체 화면",
   "Exit Full Screen": "전체 화면 나가기",
-  "More Options": "추가 옵션"
+  "More Options": "추가 옵션",
+  "Audio": "Audio",
+  "Subtitles": "Subtitles"
```
Comment (pilievOoyala) [inline languageFiles/ko.json]: You are missing 'Undefined language' here

**Comment by pilievOoyala** [comment]
Have you written up a ticket for the Docs team to translate these?

**Comment by aeng7** [comment]
Should we wait until we receive the translations to merge these? Or will merging this help/unblock development?

**Comment by pilievOoyala** [comment]
@aeng7 We are going to need this. We can do another pull request when the translated string are in. Otherwise it's not going to labeled at all because our localizer will return empty string.

**Comment by pilievOoyala** [comment]
@yaroslavgribov @KseniiaSokolovaEpam  Is there a ticket for the Docs team to localize this? Please respond with the number. If not please create one and link that.

**Comment by pilievOoyala** [comment]
Otherwise LGTM.  @aeng7  can you review too?

**Comment by pilievOoyala** [comment]
I actually created one. https://jira.corp.ooyala.com:8443/browse/DOC-3774.  @KseniiaSokolovaEpam  @yaroslavgribov  Please let me know if you already created one. So we can close one of them.

**Comment by aeng7** [comment]
LGTM

**Pull request merged**